### PR TITLE
{bio}[intel/2018a] IQ-TREE v1.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c6c445b04c42626240aad60c4c5f2476 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/25e6e15d9b817296dee3020e247f8571 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/78f3fd2f891e8a64686be35609452d80 for a full test report.

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.6.6-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+toolchainopts = {'usempi': True}
+
+github_account = 'Cibiv'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['31bc44e3a79edb889f3e2ed396d1492b8e38d93437b2ae64deb452867929dc56']
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('Eigen', '3.3.4', '', True),
```
Comment (vanzod) [inline easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.6.6-intel-2018a.eb]: Is there a particular reason we need to use `Eigen` built with `dummy`?

Comment (boegel) [reply easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.6.6-intel-2018a.eb]: @vanzod `Eigen` is only header files, there's absolutely no point in using anything else than `dummy`

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**